### ISSUE TITLE
[meta] revisit scopes

### ISSUE THREAD

**Issue by escapewindow** [mozilla-releng/scriptworker-scripts#170]
Tl;dr, let's limit our usage of scopes to the restricted set, and determine non-restricted behavior/format/action based on `task.payload`.

Details:

Our usage of scopes came from the original funsize [signingworker](https://github.com/mozilla-releng/signingworker/), which used scopes to determine the type of cert (dep/nightly/release), as well as the signing format (gpg/authenticode/dmg etc). With CoT, we restricted the `cert` scope, allowing for dep but restricting nightly and release to certain trees. Over time, we found that using scopes for formats was unwieldy, without seeing any benefit. We moved to specifying format in the task payload, and only setting the cert scope.

Let's continue this pattern throughout the scripts. Restricted scope types (e.g., balrog server, signing cert level, beetmover bucket) should continue to be scopes, and we should continue restricting them in `scriptworker.constants.DEFAULT_CONFIG.cot_restricted_scopes`. Any other scopes, we should deprecate, and move those settings into the task payload.

(To clarify: in the current proposal, all of `cert:dep-signing`, `cert:nightly-signing`, and `cert:release-signing` would each be scopes, even though only the latter 2 are restricted. We're just talking about removing the `format` scopes, for signing.)

Let's resolve this issue once all scripts follow this pattern.

**Comment by tomprince** [comment]
I'll note (as partial motivation) that essentially all the non-restricted scopes are granted to level-1 repos to enable staging releases.

One possible variant would be to have things defined in the payload, but also require scopes when used on level-3 workers (this might make sense for the balrog `:channel:` scopes, if those had any effect).
